01.fit_a_line/README.md

```diff
@@ -116,49 +117,58 @@ When training complex models, we usually have one more split: the validation set
 
 `fit_a_line/trainer.py` demonstrates the training using [PaddlePaddle](http://paddlepaddle.org).
 
-### Initialize PaddlePaddle
+### Datafeeder Configuration
```

Comment:
All these sections including training and inference are under ##Dataset section, should we bring these 1 level up?